### PR TITLE
Clearer config

### PR DESCRIPTION
**Pull request by ravenbuilder934**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
- Improves clarity of config descriptions
- Removes old, dead config options
-

Review please

### CODE REVIEW

**Comment by Raycoms** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/ldtteam/minecolonies/pull-request/6318/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ldtteam/minecolonies)

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 1]
```diff
@@ -65,35 +65,29 @@
...
   "minecolonies.config.allowinfinitesupplychests.comment": "Should players be able to place an infinite amount of supply camps/ships?",
   "minecolonies.config.allowinfinitecolonies": "Infinite Colonies",
   "minecolonies.config.allowinfinitecolonies.comment": "Should players be allowed to abandon their colony to create a new one easily? Note: This is buggy!",
   "minecolonies.config.allowotherdimcolonies": "Other Dim Colonies",
-  "minecolonies.config.allowotherdimcolonies.comment": "Should colonies in other dimensions be allowed? Note: This is buggy if the dimension doesn't have day/night!",
-  "minecolonies.config.citizenrespawninterval": "Citizen Respawn Interval",
-  "minecolonies.config.citizenrespawninterval.comment": "Average citizen respawn interval (in seconds).",
+  "minecolonies.config.allowotherdimcolonies.comment": "Should colonies in other dimensions be allowed?",
+  "minecolonies.config.citizenrespawninterval": "Inital Citizen Respawn Interval",
+  "minecolonies.config.citizenrespawninterval.comment": "If a citizen dies and there are less than four citizens left, this changes the amount of time before another citizen spawns.",
```
Comment (gattsuru) [inline src/main/resources/assets/minecolonies/lang/en_us.json]: Not sure if it's worth spelling out, but citizensrespawninterval applies for all (sane) settings of initialcitizenamount, not just the default four.

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 2]
```diff
@@ -104,15 +98,15 @@
...
+  "minecolonies.config.holidayfeatures.comment": "Should special holiday content be displayed? (Santa hats around Christmas time, semi-transparent citizens near Halloween, etc.)",
+  "minecolonies.config.updaterate": "Update Rate",
   "minecolonies.config.updaterate.comment": "AI Update rate. Increase this to improve performance. This throttles citizen updates.",
-  "minecolonies.config.dirtfromcompost": "Compost->Dirt",
+  "minecolonies.config.dirtfromcompost": "Compost to Dirt",
   "minecolonies.config.dirtfromcompost.comment": "Quantity of dirt per compost filling.",
-  "minecolonies.config.luckyblockchance": "Lucky-Blocks",
-  "minecolonies.config.luckyblockchance.comment": "Chance to get a lucky block as a percentage.",
+  "minecolonies.config.luckyblockchance": "Ore Chance",
+  "minecolonies.config.luckyblockchance.comment": "Chance for the Miner to get an ore when mining cobblestone or stone. This is a percentage.",
```
Comment (gattsuru) [inline src/main/resources/assets/minecolonies/lang/en_us.json:109]: Maybe emphasize that it's a bonus ore?  I dun't expect it to happen often, but could see someone think this limits even the miner's ability to gather worldgen ores.

**Review by gattsuru — COMMENTED** [review]
Thanks a ton for doing this.  Couple pedantic nitpicks.

"LuckyBlocks" can be set by json data packs, so it may be better to be vague about what material the LuckyBlock modifier applies to.

File: src/main/resources/assets/minecolonies/lang/en_us.json (hunk 1 above)
Comment (ravenbuilder934) [reply src/main/resources/assets/minecolonies/lang/en_us.json]: `  "minecolonies.config.citizenrespawninterval.comment": "If a citizen dies and there are less than the initial number of citizens left, this changes the amount of time before another citizen spawns.",` maybe? Bit wordy, hmm.

Comment (gattsuru) [reply src/main/resources/assets/minecolonies/lang/en_us.json]: Wordy, but probably clearer than referencing another variable directly.  And it does end up spaced apart by the configbuilder.  If you want to snip it down, this does effect the first citizens spawn rate as well, so : 

`"minecolonies.config.citizenrespawninterval.comment": "If a colony is under the initial citizen count, this changes the amount of time before a new citizen spawns.", `

Comment (ravenbuilder934) [reply src/main/resources/assets/minecolonies/lang/en_us.json]: Oh first citizen spawn rate too? I'll go with that then

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 3]
```diff
@@ -104,65 +98,57 @@
   "minecolonies.config.allowplayerschematics": "Player Schematics",
   "minecolonies.config.allowplayerschematics.comment": "Should player-made schematics be allowed",
   "minecolonies.config.allowglobalnamechanges": "Name Changes",
-  "minecolonies.config.allowglobalnamechanges.comment": "Should players be allowed to change names? -1 for false, 0 for specific groups, 1 for true.",
+  "minecolonies.config.allowglobalnamechanges.comment": "Should players be allowed to change citizen names? -1 for false, 0 for certain players only, 1 for true. (To set which players can change the names, see \"Special Permissions Group\" below.)",
   "minecolonies.config.holidayfeatures": "Holiday Features",
-  "minecolonies.config.holidayfeatures.comment": "Should special holiday content be displayed?",
-  "minecolonies.config.updaterate": "Update-Rate",
+  "minecolonies.config.holidayfeatures.comment": "Should special holiday content be displayed? (Santa hats around Christmas time, semi-transparent citizens near Halloween, etc.)",
```
Comment (someaddons) [inline src/main/resources/assets/minecolonies/lang/en_us.json]: spoiler!

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 4]
```diff
@@ -104,65 +98,57 @@
...
-  "minecolonies.config.canplayeruseaddofficercommand.comment": "Should the player be allowed to use the /mc colony addOfficer command?",
-  "minecolonies.config.canplayerusedeletecolonycommand": "Can a Player use Delete Colony Command",
-  "minecolonies.config.canplayerusedeletecolonycommand.comment": "Should the player be allowed to use the /mc colony delete command?",
+  "minecolonies.config.canplayerusertpcommand": "Can Players Use the RTP Command",
+  "minecolonies.config.canplayerusertpcommand.comment": "Should players be allowed to use the /mc rtp command?",
+  "minecolonies.config.canplayerusecolonytpcommand": "Can Players Use the Colony TP Command",
+  "minecolonies.config.canplayerusecolonytpcommand.comment": "Should players be allowed to use the /mc colony teleport command?",
+  "minecolonies.config.canplayeruseallytownhallteleport": "Can Players TP to Allies",
+  "minecolonies.config.canplayeruseallytownhallteleport.comment": "Can players teleport to allied colonies?",
+  "minecolonies.config.canplayerusehometpcommand": "Can Players Use the Home TP Command",
```
Comment (someaddons) [inline src/main/resources/assets/minecolonies/lang/en_us.json:130]: should probably mention that one is owner-only

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 5]
```diff
@@ -183,9 +169,9 @@
   "minecolonies.config.spawnbarbariansize.comment": "The min size of a barbarian horde.",
   "minecolonies.config.maxbarbariansize": "Max Barbarian Size",
   "minecolonies.config.maxbarbariansize.comment": "The max size of a barbarian horde.",
-  "minecolonies.config.dobarbariansbreakthroughwalls": "Do Barbarians Break through Walls",
-  "minecolonies.config.dobarbariansbreakthroughwalls.comment": "Whether or not barbarians can break, scale, and bridge obstacles.",
-  "minecolonies.config.shouldraiderbreakdoors": "Should Raiders Break Doors",
+  "minecolonies.config.dobarbariansbreakthroughwalls": "Do Raiders ",
```
Comment (someaddons) [inline src/main/resources/assets/minecolonies/lang/en_us.json]: sth is missing here

Comment (ravenbuilder934) [reply src/main/resources/assets/minecolonies/lang/en_us.json]: Heh heh, oops.

**Review by Mekle001 — APPROVED** [review]

**Review by someaddons — APPROVED** [review]

**Pull request merged**